app/views/orga/seasons/show.html.slim~

```diff
@@ -0,0 +1,18 @@
+= render 'orga/breadcrumbs'
```

Comment:
This seems like a backup file, automatically created by your editor. Can you please remove this from the PR again? Hint: it's always best to select/stage the files for a commit individually; avoid using `git add .` which will also add unwanted backup files like this one.